include/MainWindow.h

```diff
@@ -55,10 +56,7 @@ class MainWindow : public QMainWindow
 {
 	Q_OBJECT
 public:
-	QMdiArea* workspace()
-	{
-		return m_workspace;
-	}
```

Comment:
You didn't have to move this inside the .cpp file, but its fine with me.

Reply:
I thought that `static_cast<>` needs the definition of the `MovableQMdiArea` class which is not known at that point, but it is available in the `.cpp` file (since the preprocessor copies the header to where the `#include "MainWindow.h"` line is).